lib/store/authenticator_db.dart

```diff
@@ -25,6 +26,18 @@ class AuthenticatorDB {
   }
 
   Future<Database> _initDatabase() async {
+    if (Platform.isWindows || Platform.isLinux) {
+      var databaseFactory = databaseFactoryFfi;
+      return await databaseFactory.openDatabase(
+        (await getDownloadsDirectory())!
```

Comment:
Can you confirm what the path will look like on windows or linux?

Reply:
- {%USERPROFILE}\.ente.authenticator.db
- {%USERPROFILE}\.ente.offline_authenticator.db

Windows: UserProfile is the User's Profile (The folder above Downloads folder)
Linux: UserProfile is User's home folder (same as above)